src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -64,10 +64,15 @@ object ScalafixPlugin extends AutoPlugin {
         "Optional list of artifacts to resolve to run custom rules. " +
           "Can be set in ThisBuild or at project-level."
       )
+    @deprecated(
+      "scalafixScalaBinaryVersion now follows scalaBinaryVersion by default",
+      "0.12.1"
+    )
     val scalafixScalaBinaryVersion: SettingKey[String] =
       settingKey[String](
-        "The Scala binary version used for scalafix execution. Can be set in ThisBuild or at project-level. " +
-          "Custom rules must be compiled against that binary version. Defaults to 2.12."
+        "The Scala binary version used for scalafix execution. Must be set at project-level. " +
```

Comment:
Until [recently](https://github.com/scalacenter/sbt-scalafix/pull/380), `ThisBuild` was the only place where `scalafixScalaBinaryVersion` could be set, so most of the clients must be setting it at that level currently. However, that should not be a problem because the only reason for them to set it was to follow `scalaScalafixBinaryVersion`, which is now the default (at project-level, thus the inability to override it at `ThisBuild` level).

That's why the key is being deprecated, and the change in where it effectively not a breaking change.